recipes/r-winch/meta.yaml

```diff
@@ -0,0 +1,94 @@
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dl'   # [win]
+
+requirements:
+  build:
+    - {{ stdlib("c") }}  # [not win]
+    - {{ stdlib("m2w64_c") }}  # [win]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-lifecycle
+    - r-procmaps >=0.0.2
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-lifecycle
+    - r-procmaps >=0.0.2
+
+test:
+  commands:
+    - $R -e "library('winch')"           # [not win]
+    - "\"%R%\" -e \"library('winch')\""  # [win]
+
+about:
+  home: https://r-prof.github.io/winch/
+  dev_url: https://github.com/r-prof/winch
+  license: GPL-3.0-only
+  summary: Obtain the native stack trace and fuse it with R's stack trace for easier debugging
+    of R packages with native code.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - thewchan
+
+# Package: winch
+# Title: Portable Native and Joint Stack Traces
+# Version: 0.1.1
+# Date: 2024-02-19
+# Authors@R: c(person(given = "Kirill", family = "Muller", role = c("aut", "cre"), email = "kirill@cynkra.com", comment = c(ORCID = "0000-0002-1416-3412")), person(given = "R Consortium", role = "fnd"), person(given = "Ian Lance", family = "Taylor", role = "aut", comment = "Bundled libbacktrace library"), person(given = "Free Software Foundation", role = "cph", comment = "Bundled libbacktrace library"))
+# Description: Obtain the native stack trace and fuse it with R's stack trace for easier debugging of R packages with native code.
+# License: GPL-3
+# URL: https://r-prof.github.io/winch/, https://github.com/r-prof/winch
+# BugReports: https://github.com/r-prof/winch/issues
+# Imports: lifecycle, procmaps (>= 0.0.2)
+# Suggests: DBI, knitr, magrittr, purrr, rlang (>= 0.4.8), rmarkdown, RSQLite, testthat (>= 3.0.0), vctrs
+# VignetteBuilder: knitr
+# Encoding: UTF-8
+# Biarch: yes
+# RoxygenNote: 7.3.1
+# Config/testthat/edition: 3
+# NeedsCompilation: yes
+# Packaged: 2024-02-19 07:58:53 UTC; kirill
+# Author: Kirill Muller [aut, cre] (<https://orcid.org/0000-0002-1416-3412>), R Consortium [fnd], Ian Lance Taylor [aut] (Bundled libbacktrace library), Free Software Foundation [cph] (Bundled libbacktrace library)
```

Comment:
This package is vendoring `libbacktrace` we have this available on Conda Forge (`backtrace`). This recipe should either rework to use the Conda Forge version (preferred) or at minimum make sure to clearly package the license for `libbacktrace` with the artifact.